spec.bs

```diff
@@ -676,3 +676,190 @@ To <dfn>determine the numeric value</dfn> of a [=signal base value=]
 {{double}}.
 
 Issue: Fill in.
+
+Privacy considerations {#privacy-considerations}
+================================================
+
+<em>This section is non-normative.</em>
+
+Cross-site information disclosure {#cross-site-information-disclosure}
+----------------------------------------------------------------------
+
+This API lets isolated contexts with access to cross-site data (i.e. <a
+href="https://wicg.github.io/shared-storage/">Shared Storage</a> worklets/<a
+href="https://wicg.github.io/turtledove/">Protected Audience</a> script runners)
+send aggregatable reports over the network.
+
+Aggregatable reports contain encrypted high entropy cross-site information, in
+the form of key-value pairs (i.e. contributions to a histogram). The information
+embedded in the contributions is arbitrary but can include things like browsing
+history and other cross-site activity. The API aims to provide some protection
```

Comment:
"some protection" sounds weaker than what I think this API actually provides

Reply:
Rephrased this to "aims to protect"